src/AppBundle/Entity/Relay.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+namespace AppBundle\Entity;
+
+use DateTime;
+use Doctrine\ORM\Mapping as ORM;
+use Symfony\Bridge\Doctrine\Validator\Constraints\UniqueEntity;
+
+/**
+ * A contributor can relay a notice from another contributor. This entity represents this relation.
+ *
+ * @ORM\Entity
+ * @ORM\Table(name="relay")
+ * @UniqueEntity(
+ *     fields={"contributor", "notice"},
+ *     errorPath="contributor",
+ *     message="This contributor already relayed that notice."
+ * )
+ */
+class Relay
+{
+    /**
+     * @var Contributor
+     *
+     * @ORM\ManyToOne(targetEntity=Contributor::class, inversedBy="relayedNotices", fetch="EAGER")
```

Comment:
Might change the field name to `relayed_by`  too.

Reply:
Yes, I pushed too fast but the tests got by back :sweat_smile: 

Fixed !

Reply:
I don't know if you chose the name `relayed_by_id` for the db field, but this is the name in the migration, might be good idea to enforce this name here. By the way, `relayed_by` seems sufficient name from my point of view.

Reply:
> I don't know if you chose the name relayed_by_id for the db field, but this is the name in the migration, might be good idea to enforce this name here. By the way, relayed_by seems sufficient name from my point of view.

You didn't answer on this one.